packages/web-react/src/components/CheckboxField/stories/CheckboxField.tsx

```diff
@@ -5,14 +5,6 @@ import { SpiritCheckboxFieldProps } from '../../../types';
 
 const Story: ComponentStory<typeof CheckboxField> = (args: SpiritCheckboxFieldProps) => <CheckboxField {...args} />;
 
-Story.args = {
-  isChecked: true,
-  isDisabled: false,
-  isItem: false,
-  isLabelHidden: false,
-  isRequired: false,
-  label: 'Label',
-  name: 'example',
-};
+Story.args = {};
```

Comment:
Must be this line here? Can we remove it? Or something breaks if we do that?

Reply:
It doesn't matter, but I'd rather keep it here because these arguments somehow belong to the stories even if the object is empty. Even when I went through it, I occasionally came across an empty one, honestly I probably wouldn't see a problem with it.

Reply:
Does it break something if we remove this? -> Leave it and add a comment about why this must be here.
No? -> Remove it because there should not be kept unnecessary code.

Yes, there are these lines in other stories but that is not a reason to keep them. Please check if removing has an impact on something. If not, remove it. Thanks.

Reply:
Removal should have no negative effect

Reply:
done